app/Category.php

```diff
@@ -6,5 +6,17 @@
 
 class Category extends Model
 {
-    //
+    protected $guarded = ['id'];
+    protected $fillable = [
+        'title',
+    ];
+    public function learnedLesson()
+    {
+        return $this->hasMany(learnedLesson::class);
```

Comment:
i think basing from ER diagram i gave to you, category has only relationship with lesson. please reconsider this relationship and based it from ER Diagram

Reply:
-I changed the name for my convinience. I'll change the name.